wolfssl/error-ssl.h

```diff
@@ -167,6 +167,7 @@ enum wolfSSL_ErrorCodes {
     CLIENT_CERT_CB_ERROR         = -436,   /* Client cert callback error */
     SSL_SHUTDOWN_ALREADY_DONE_E  = -437,   /* Shutdown called redundantly */
     TLS13_SECRET_CB_E            = -438,   /* TLS1.3 secret Cb fcn failure */
+    DTLS_SIZE_ERROR              = -439,   /* Trying to send too much data */
```

Comment:
Does this need to be added internal.c `wolfSSL_ERR_reason_error_string`?

Reply:
It is in the function.